web/src/components/Plugin/UI/cors.tsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import type { FormInstance } from 'antd/es/form';
+import { Button, Form, Input, InputNumber, Switch } from 'antd';
+import { MinusCircleOutlined, PlusOutlined } from '@ant-design/icons';
+import { useIntl } from '@/.umi/plugin-locale/localeExports';
+
+type Props = {
+  form: FormInstance;
+  ref?: any;
+};
+
+const FORM_ITEM_LAYOUT = {
+  labelCol: {
+    span: 6,
+  },
+  wrapperCol: {
+    span: 8
+  },
+};
+
+export const FORM_ITEM_WITHOUT_LABEL = {
+  wrapperCol: {
+    sm: { span: 8, offset: 6 },
+  },
+};
+
+const Cors: React.FC<Props> = ({ form }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Form
+      form={form}
+      {...FORM_ITEM_LAYOUT}
+    >
+      <Form.Item
+        name="allow_origins"
+        label="allow_origins"
+        initialValue={"*"}
+      >
+        <Input></Input>
+      </Form.Item>
+      <Form.Item
+        name="allow_methods"
+        label="allow_methods"
+        initialValue={"*"}
+      >
+        <Input></Input>
```

Comment:
```suggestion
        <Input />
```

Reply:
`Input` is not good here, use `Multiple Select`, please refer to Route create page.

Reply:
![image](https://user-images.githubusercontent.com/2106987/114296216-a106eb80-9adc-11eb-8584-76032e346ca5.png)